zh-CN/mapping_dev.zh_CN

```diff
@@ -223,25 +223,25 @@ dlgScJiraCommitMessageSelect.col"Fix Version"=修复版本
 dlgScJiraCommitMessageSelect.col"Key"=键
 dlgScJiraCommitMessageSelect.col"Status"=状态
 dlgScJiraCommitMessageSelect.col"Summary"=摘要
-dlgScJiraCommitMessageSelect.hdl=按JIRA问题选择提交消息
+dlgScJiraCommitMessageSelect.hdl=按 JIRA 问题选择提交消息
 dlgScJiraCommitMessageSelect.inf=选定的问题摘要将用作提交消息。
 dlgScJiraCommitMessageSelect.lbl"Enter the JQL-Queries which will be loaded in the specified order: every query on a new line, \$\{projects\} will be substituted by the JIRA project key\(s\) which is derived or explicitly specified in your .gitbugtraq file.\n\nYou can compose a JQL query directly in JIRA, using Issues\|Search and go to Advanced view there."=\
- 输入将按指定顺序加载的JQL查询：新行上的每个查询， \$\{projects\}将替换为.gitbugtraq文件中派生或明确指定的JIRA项目密钥。\r\n您可以直接在JIRA中编写JQL查询，使用 问题|搜索 并转到高级视图。
+ 输入将按指定顺序加载的 JQL 查询：新行上的每个查询，\$\{projects\} 将替换为 .gitbugtraq 文件中派生或明确指定的 JIRA 项目密钥。\r\n您可以直接在 JIRA 中编写 JQL 查询，使用问题|搜索并转到高级视图。
 dlgScJiraCommitMessageSelect.tle=选择问题
 dlgScJiraResolveIssue.btn"Don't Resolve"=不解决
 dlgScJiraResolveIssue.btn"Resolve"=解决
-dlgScJiraResolveIssue.edt"Resolution"=Resolution
+dlgScJiraResolveIssue.edt"Resolution"=解决
 dlgScJiraResolveIssue.edt"Summary"=摘要
-dlgScJiraResolveIssue.hdl%1=解决问题$1
+dlgScJiraResolveIssue.hdl%1=解决问题 $1
 dlgScJiraResolveIssue.inf=选择是否解决此问题以及要标记为已解决的版本。
-dlgScJiraResolveIssue.tle=解决JIRA问题
+dlgScJiraResolveIssue.tle=解决 JIRA 问题
 dlgScMasterPasswordChange.edt"Current Master Password"=当前主密码
 dlgScMasterPasswordChange.edt"New Master Password"=新主密码
 dlgScMasterPasswordChange.edt"Retype New Master Password"=再次输入
 dlgScMasterPasswordChange.hdl=更改或重置主密码
 dlgScMasterPasswordChange.inf=要更改主密码，请输入当前密码。要没有主密码，请将字段留空。
 dlgScMasterPasswordChange.lbl"When setting a new master password, all stored passwords and passphrases will be lost and need to be re-entered when required!"=\
- 设置新的主密码时，所有存储的密码和口令都将丢失，需要时需要重新输入！
+ 设置新的主密码时，所有存储的密码和密码都将丢失，需要时需要重新输入！
```

Comment:
I don't think it needs to be modified.. Please check again.

Reply:
Thanks, I've fixed it.